README.md

```diff
@@ -147,8 +147,40 @@ sudo dpkg -i postgresql-9.3-tsearch-extras_0.1.2_amd64.deb
 sudo apt-get install postgresql-9.4
 wget https://dl.dropboxusercontent.com/u/283158365/zuliposs/postgresql-9.4-tsearch-extras_0.1_amd64.deb
 sudo dpkg -i postgresql-9.4-tsearch-extras_0.1_amd64.deb
+```
+
+On Fedora 22:
 
-# Then, all versions:
+```
+sudo dnf install libffi-devel memcached rabbitmq-server openldap-devel python-devel redis postgresql-server postgresql-devel postgresql libmemcached-devel
+wget https://launchpad.net/~tabbott/+archive/ubuntu/zulip/+files/tsearch-extras_0.1.3.tar.gz
+tar xvzf tsearch-extras_0.1.3.tar.gz
+cd ts2
+make
+sudo make install
+
+# now create the missing dictionary files
+sudo touch /usr/share/pgsql/tsearch_data/english.stop
+sudo touch /usr/share/pgsql/tsearch_data/en_us.dict
+sudo touch /usr/share/pgsql/tsearch_data/en_us.affix
+
+# Edit the postgres settings:
+sudo vi /var/lib/pgsql/data/pg_hba.conf
+
+# Add these two lines after line 80:
+local   zulip           zulip                                   md5
+host    all             all             127.0.0.1/32            md5
+
+# Start the services
+sudo service redis start
```

Comment:
CentOS 7 and Fedora 22 use systemd as the init system. Although the service syntax is still supported, maybe it's better to support proper, systemctl syntax for managing services?

In this case, that would be
```
systemctl start redis
```